static/css/theme-toggle.css

```
@@ -0,0 +1,264 @@
+          left: unset;
+          right: unset;
+          top: unset;
+          bottom: unset;
+          margin: unset;
+          padding: unset;
+          position: relative;
+          visibility: hidden;
+          z-index: 9;
+      }
+
+      .theme-control {
+          position: relative;
+          z-index: 0;
+      }
+
+      .theme-control::before {
+          content: "";
+          height: 1.1rem;
+          width: 2rem;
+          border-radius: 3rem;
+          background: #000;
+          position: absolute;
+          top: 50%;
+          left: 50%;
+          transform: translate(-50%, -50%);
+          display: block;
+          visibility: visible;
+          z-index: 0;
+      }
+
+
+      .theme-control::after {
+          border-radius: 50%;
+          left: 0;
+          right: 0;
+          top: 0;
+          bottom: 0;
+          top: 50%;
+          left: 50%;
+          content: '';
+          background: #0071b5;
+          width: 1rem;
+          height: 1rem;
+          display: block;
+          position: absolute;
+          transform: translate(-0.85rem, -50%);
+          visibility: visible;
+          -moz-transition: transform 350ms;
+          -webkit-transition: transform 350ms;
+          transition: transform 350ms;
+          z-index: 0;
+
+      }
+
+      .theme-control:has(label > input:checked)::after {
+          transform: translate(-0.15rem, -50%);
+      }
+  }
+
+  body[data-theme="light"] {
+
+      img#logo {
+          filter: unset;
+      }
+  }
+
+  body[data-theme="dark"] {
+      background: #333;
+      color: #fff;
+
+      label::before,
+      label::after {
+          background-color: #333;
+      }
+
+      label::before {
+          background: url('../images/sun-light.svg') no-repeat center center / cover;
+      }
+
+      label::after {
+          background: url('../images/moon-light.svg') no-repeat center center / cover;
+      }
+
+
+      div#sidebar {
+          background: #444;
+          border-right: 1px solid #666;
+          box-shadow: 0 0 20px #555;
+          -webkit-box-shadow: 0 0 20px #555;
+          -moz-box-shadow: 0 0 20px #555;
+      }
+
+      a.toc_title,
+      a.toc_title:visited {
+          color: #f4f4f4;
+      }
+
+      a.toc_title:hover {
+          text-decoration: underline;
+      }
+
+      .toc_section li a {
+          text-decoration: none;
+          color: #f4f4f4;
+      }
+
+      img#logo {
+          filter: brightness(0) invert(1);
+      }
+
+      div.warning {
+          color: #CC6060;
+      }
+
+      a,
+      a:visited {
+          color: #f4f4f4;
+      }
+
+      a:active,
+      a:hover {
+          color: #ffffff;
+      }
+
+      table .rule {
+          background: #777;
+      }
+
+      tt {
+          background: #444;
+          border: 1px solid #777;
+      }
+
+      pre {
+          border-left: 5px solid #aaa;
+      }
+  }
+
+  @media screen and (prefer-color-scheme: light) {
```

typo (though I don't think you need this query)

```suggestion
  @media screen and (prefers-color-scheme: light) {
```